PWAWidgets/README.md

```diff
@@ -690,9 +690,13 @@ Developers will use `removeByTag()` to remove all Instances of a Widget. This me
 
 ## Widget-related Events
```

Comment:
The way this section is broken up feels weird now that each widget event is unique. What do you think about breaking it up into:
- Widget-related Events
  - WidgetEvent
    - Describe properties
    - List subtypes: install/uninstall/resume
    - Steps for creating a WidgetEvent
    - Describe each subtype
  - WidgetClickEvent
    - Note that it extends WidgetEvent and describe its additional properties  
    - Sample code
    - Steps for creating a WidgetClickEvent
    
I'm also open to this being a follow up change so it's easier to read the diffs